src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -193,16 +193,16 @@ export class ResultsViewPageStore {
         }
     });
 
-    readonly molecularData = remoteData({
+    readonly molecularData = remoteData<GeneMolecularData[]>({
         await: () => [
             this.studyToDataQueryFilter,
             this.genes,
             this.selectedMolecularProfiles
         ],
-        invoke: async() => {
+        invoke: () => {
```

Comment:
why take away async?

Reply:
for code legibility, since nothing async is actually happening

Reply:
we do call api here though.  215

Reply:
but we're returning the promise, theres no awaiting going on inside